pkg/internal/istio/BUILD.bazel

```
@@ -6,7 +6,7 @@ go_library(
         "istio.go",
         "istio_deepcopy.gen.go",
     ],
-    importpath = "github.com/jetstack/cert-manager/pkg/issuer/acme/http/internal/istio",
+    importpath = "github.com/jetstack/cert-manager/pkg/internal/istio",
     visibility = ["//:__subpackages__"],
```

```suggestion
    visibility = ["//pkg/:__subpackages__"],
```
This visibility should be set to `//pkg/:__subpackages__`, so it stays the same as the go build tools.